lib/bin/lwm2m_carrier/doc/CHANGELOG.rst

```diff
@@ -5,6 +5,34 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+liblwm2m_carrier 0.9.0
+**********************
+
+Snapshot release for modem firmware version 1.2.0 and the |NCS| 1.3.0.
+
+This release is intended to let users begin the integration on the At&T device management platform and start the certification process with AT&T.
+
+Certification status
+====================
+
+The library is not certified with any carrier.
+
+Changes
+=======
+
+* Added new APIs to support portfolio objects. A new portfolio object instance can be created using ``lwm2m_carrier_portfolio_instance_create()``.
```

Comment:
There is only one object, but multiple instances Maybe rephrase the first sentence to something like: `Added new APIs to create and access Portfolio object instances`? I should have changed this earlier, sorry.